template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/feature_engineering/notebooks/GenerateAndWriteFeatures.py.tmpl

```diff
@@ -116,22 +116,20 @@ features_df = compute_features_fn(
 # COMMAND ----------
 # DBTITLE 1, Write computed features.
 
-from databricks import feature_store
-
-fs = feature_store.FeatureStoreClient()
-
+from databricks.feature_engineering import FeatureEngineeringClient
```

Comment:
We should only do this if UC is enabled, you can see other examples where we use:
```{{ if (eq .input_include_models_in_unity_catalog `no`) }}```, or does this client work for non-UC cases as well?

Reply:
OK -- I think need to update with conditionals. I was wondering where they were but they need to be added!